webextensions/api/scripting.json

```diff
@@ -0,0 +1,393 @@
+{
+  "webextensions": {
+    "api": {
+      "scripting": {
+        "ContentScriptFilter": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/ContentScriptFilter",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "edge": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox_android": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "opera": {
+                "version_added": "74",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "InjectionResult": {
```

Comment:
InjectionResult doesn't need an entry either as it's part of `executeScript`, and in fact the latest version of https://github.com/mdn/content/pull/14725 doesn't list the article either.

Reply:
Removed